docker-compose-dev.yaml

```diff
@@ -0,0 +1,133 @@
+version: '3.7'
+
+services:
+  nginx:
+    build: ./nginx
+    image: djangofiles-nginx:${VERSION:-latest}
+    deploy:
+      mode: global
+      labels:
+        - "traefik.enable=true"
+        - "traefik.docker.network=traefik-public"
+        - "traefik.constraint-label=traefik-public"
+        - "traefik.http.routers.STACK_NAME-http.rule=Host(${TRAEFIK_HOST:-``})"
+        - "traefik.http.routers.STACK_NAME-http.entrypoints=http"
+        - "traefik.http.routers.STACK_NAME-http.middlewares=https-redirect"
+        - "traefik.http.routers.STACK_NAME-https.rule=Host(${TRAEFIK_HOST:-``})"
+        - "traefik.http.routers.STACK_NAME-https.entrypoints=https"
+        - "traefik.http.routers.STACK_NAME-https.tls=true"
+        - "traefik.http.services.STACK_NAME.loadbalancer.server.port=80"
+        - "traefik.http.services.STACK_NAME.loadbalancer.server.scheme=http"
+    volumes:
+      - static_dir:/data/static
+      - media_dir:/data/media
+      - ${APP_FILES:-./app}:/app
+#    secrets:
+#      - source: cssnr_basic_http_auth
+#        target: /etc/nginx/auth.users
+    depends_on:
+      - app
+#      - flower
+#      - redis-commander
+#      - phpmyadmin
+    ports:
+      - "80:80"
+
+  app:
+    build: ./app
+    image: djangofiles-app:${VERSION:-latest}
+    env_file: settings.env
+    command: "python manage.py runserver_plus 0.0.0.0:9000"
+    healthcheck:
+      test: ["CMD", "curl", "-sf", "http://localhost:9000/app-health-check/"]
```

Comment:
Make this:
    command: >
      sh -c "python manage.py migrate &&
      python manage.py runserver_plus 0.0.0.0:9000"